src/scipp/html/formatting_html.py

```diff
@@ -33,7 +34,7 @@ def _format_array(data, size, ellipsis_after, do_elide=True):
             else:
                 s.append(f'{{dims=[{dims}], unit={elem.unit}, coords=[{coords}]}}')
         else:
-            s.append(str(elem))
+            s.append(value_to_string(elem))
```

Comment:
Is the default precision causes width issues? It looks like it is currently defaulting to 3, but it used to be 2 here?

Reply:
Don't know. It is fine on my monitor even when the window is only half the monitor width.

Reply:
One issue with `value_to_string` is that if you have a value like `2.0e-6`, it will print `2.000e-6`, with unnecessary zeros. Maybe we make it better?

Reply:
I feel 2.000e-6 is better, since if I just see 2.0e-6 I'll think it is rounded to one decimal place?

Reply:
Might be nice if we could also render the values in other dtypes, such as `vector3`, with a limited precision.
![Screenshot at 2022-01-05 10-55-07](https://user-images.githubusercontent.com/39047984/148198190-e12ee061-5966-4763-b3b6-670a0517c1f5.png)

Reply:
Since this is unrelated, I'll approve.